### PR TITLE
adding hamish's photos

### PR DESCRIPTION
**Pull request by hamishf-bgl**

### CODE REVIEW

File: src/config/images.json [hunk 1]
```diff
@@ -17,6 +17,18 @@
   {
     "src": "https://farm1.staticflickr.com/862/43836017002_e6af423af3_b.jpg" 
   },
+  {
+    "src": "https://farm2.staticflickr.com/1776/42081584650_a7b778860f_o.png"
```
Comment (rkclark) [inline src/config/images.json]: needs comma on the end :)

File: src/config/images.json [hunk 2]
```diff
@@ -17,6 +17,18 @@
...
+    "photographer": "Hamish Forbes"
+  },
+  {
+    "src": "https://farm2.staticflickr.com/1795/42081582620_23b0e2da41_o.png"
```
Comment (rkclark) [inline src/config/images.json]: needs comma on the end :)

File: src/config/images.json [hunk 3]
```diff
@@ -17,6 +17,18 @@
...
+    "photographer": "Hamish Forbes"
+  },
+  {
+    "src": "https://farm1.staticflickr.com/942/42081581090_462101bbe0_o.png"
```
Comment (rkclark) [inline src/config/images.json]: needs comma on the end :)

**Review by rkclark — CHANGES_REQUESTED** [review]

**Comment by hamishf-bgl** [comment]
Whoops! @rkclark

**Review by rkclark — APPROVED** [review]

**Pull request merged**